src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeService.kt

```diff
@@ -172,6 +172,10 @@ class NodeService(
           .stream()
           .mapToInt { el -> el.transactionAmount.value }
           .sum())
+    logger.info(
+      "Building close payment for cancellation with effective clientId: [{}], activation client is: [{}]",
+      transactionWithCancellation.clientId?.effectiveClient?.name,
```

Comment:
```suggestion
      transactionWithCancellation.clientId.effectiveClient.name,
```

Shouldn't `clientId` always be non-null?

Reply:
Yes, you're right. I fix it

Reply:
Solved by https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/319/commits/3886a7c9799cdf26e0ecd2491b069464e530fdf8